### DISCUSSION TITLE
Modifying frontend/public/locales/zh/translation.json does not work.

### DISCUSSION THREAD

**Discussion by Chenrujie-85** [kiali/kiali#7740] [Q&A] [answered]
### Describe the bug

I wanted to add a Chinese translation for kiali, so I modified translation.json, but adding a new translation to translation.json didn't work. I then tried to modify the existing translation, but it still didn't work.

before：
![开始](https://github.com/user-attachments/assets/39848129-282e-4061-abcd-b7986891aa2b)
afafter：
![之后](https://github.com/user-attachments/assets/698b6b59-9c40-4521-883b-a7061a8abbb9)

### Expected Behavior

Modify the translation.json file to add Chinese translation.

### What are the steps to reproduce this bug?

1.modify the translation.json file to add Chinese translation.
2.use import {t} from'utils/I18nUtils' where the word is used; and t(word) for translation
3.After the preceding two steps do not work, modify the existing translation in translation.json. For example, change the Chinese value of Every 10s to 每1000秒. However, the modification still does not work.

### Environment

- **Kiali version**:1.85
- **Istio version**:1.17.5
- **Kubernetes impl:** 
- **Kubernetes version**:1.25.3
- **Other notable environmental factors:**

**Comment by jmazzitelli** [comment]
> Kiali version:1.85

Kiali is currently at 1.89. I don't think we are going to be backporting new localizations to older versions. As new localizations are added/modified, they will be included the next release - so work in the master branch.

Instructions on doing i18n development work in Kiali can be found in the README here: https://github.com/kiali/kiali/blob/master/frontend/README.adoc#internationalization-i18n

**Comment by jmazzitelli** [comment]
_(converted this to a Q&A discussion as it is not a bug but related to development)_

**Comment by ferhoyos** [comment, marked as answer]
Hi @Chenrujie-85, as @jmazzitelli mentioned, it is recommended that you update Kiali to the latest version or the master branch. Some i18n improvements have been added since v1.85 (e.g., https://github.com/kiali/kiali/pull/7401), and the bug you mentioned may no longer occur.

Also note that the `t` function behaves differently depending on where it is applied. For variables outside of a React component, the `t` function only serves as an indicator for i18next-parser to include the statement in the  `translation.json` file, but it does not perform the translation. Actual translation only occurs within a React component.

**Reply by Chenrujie-85** [reply]
Thank you for your answer. I will try working on versions higher than v1.85 tomorrow

**Comment by jshaughn** [comment]
Thank you for offering to do this work, @Chenrujie-85 !
